vms/event/tests/test_shiftSignUp.py

```diff
@@ -53,116 +52,102 @@ def login_volunteer(self):
-                    volunteer_id=self.volunteer.id,
-                    shift_id=created_shift.id)), 0)
-
     def test_signup_for_same_shift_again(self):
-
         register_event_utility()
         register_job_utility()
         register_shift_utility()
 
         sign_up_page = self.sign_up_page
-        # open Shift Sign Up
+        # Open Shift Sign Up
         sign_up_page.navigate_to_sign_up()
 
-        # events shown in table
+        # Events shown in table
         with self.assertRaises(NoSuchElementException):
             sign_up_page.get_info_box()
         sign_up_page.click_to_view_jobs()
 
-        # on jobs page
+        # On jobs page
         sign_up_page.click_to_view_shifts()
 
-        # on shifts page, Sign up this shift !
+        # On shifts page, Sign up this shift !
         sign_up_page.click_to_sign_up()
 
-        # confirm on shift sign up
+        # Confirm on shift sign up
         sign_up_page.submit_form()
         with self.assertRaises(NoSuchElementException):
             sign_up_page.get_danger_box()
 
-        # sign up same shift again
-        # open Shift Sign Up
+        # Sign up same shift again
+        # Open Shift Sign Up
         sign_up_page.navigate_to_sign_up()
 
-        # events page
-        self.assertEqual(sign_up_page.get_info_box().text,sign_up_page.no_event_message)
+        # Events page
+        self.assertEqual(sign_up_page.get_info_box().text, sign_up_page.no_event_message)
 
         with self.assertRaises(NoSuchElementException):
             sign_up_page.find_table_tag()
 
     def test_empty_events(self):
-        
         register_event_utility()
         sign_up_page = self.sign_up_page
-        # open Shift Sign Up
+        # Open Shift Sign Up
         sign_up_page.navigate_to_sign_up()
 
-        # on event page
-        self.assertEqual(sign_up_page.get_info_box().text,sign_up_page.no_event_message)
+        # On event page
+        self.assertEqual(sign_up_page.get_info_box().text, sign_up_page.no_event_message)
 
         with self.assertRaises(NoSuchElementException):
             sign_up_page.find_table_tag()
             sign_up_page.click_to_view_jobs()
 
         register_job_utility()
 
-        self.assertEqual(sign_up_page.get_info_box().text,sign_up_page.no_event_message)
+        self.assertEqual(sign_up_page.get_info_box().text, sign_up_page.no_event_message)
 
         with self.assertRaises(NoSuchElementException):
             sign_up_page.find_table_tag()
 
     def test_shift_sign_up_with_outdated_shifts(self):
-
         register_event_utility()
         register_job_utility()
         sign_up_page = self.sign_up_page
 
         # create outdated shift
-        shift_1 = ["2016-05-11","9:00","15:00",6,Job.objects.get(name = 'job')]
-        s1 = create_shift_with_details(shift_1)
+        shift_1 = ["2016-05-11", "9:00", "15:00", 6, Job.objects.get(name='job')]
+        create_shift_with_details(shift_1)
```

Comment:
If something is returned, it is good practice to store the return value even if you're not using it. Someone else might want it later, but this way they'll think the function returns nothing.